src/reports/AuditPage.jsx

```diff
@@ -61,7 +72,11 @@ const AuditPage = () => {
             <TableRow key={item.id}>
               {columns.map((key) => (
                 <TableCell key={key}>
-                  {key === 'actionTime' ? formatTime(item[key], 'minutes') : item[key]}
+                  {key === 'actionTime'
+                    ? formatTime(item[key], 'minutes')
+                    : key === 'userId'
+                      ? users[item[key]] || item[key]
+                      : item[key]}
```

Comment:
This should be extracted into a function, similar to what we do for other report formatting